### ISSUE TITLE
Crash while trying to press LMB on any menu entry.

### ISSUE THREAD

**Issue by zor1984qq** [OpenRA/OpenRA#12120]
Hi! Here my crash logs in bzip2 format, I using just freshly installed current  openra-20160904-1.14.noarch package from opensuse OBS repository for my Tumbleweed Linux instance. It seems that it has implemented somehow very cutting edge new packages and despite I have already installed package called "dialog" the logs shows up errors "Could not find collection 'dialog'" while game trying to find it in my system. Also I have few mono-4.4.2 or something version errors in exceptions.log file (I wouldn't tell that the packaging team here work are bad, they doing fine but something goes wrong anyway and I have no clue what to do). Have no out of sync errors. Other issues listen on your main github.com page seems not in my case so I decide to open up a new issue. Anyway I can try to wait on actual release of your game so Its not critical for me since I still need to retrieve original game files somewhere.
[openra.logs.bzip2.zip](https://github.com/OpenRA/OpenRA/files/496147/openra.logs.bzip2.zip)
Steps for reproducing the bug:
1. Launch the game,
2. Press LMB on any menu entry then the game crashes on the desktop.

**Comment by LipkeGu** [comment]
```
Mod Chooser Mod at Version playtest-20160904
Date: 2016-09-27 16:22:23Z
Operating System: Linux (Unix 4.7.4.2)
Runtime Version: Mono 4.4.2 (Stable 4.4.2.11/f72fe45 Fri Sep  2 20:31:35 UTC 2016) CLR 4.0.30319.42000
Exception of type `System.IO.FileNotFoundException`: Could not find file "/etc/localtime"
  at System.IO.FileStream..ctor (System.String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, Boolean anonymous, FileOptions options) [0x00335] in /home/abuild/rpmbuild/BUILD/mono-4.4.2/mcs/class/corlib/System.IO/FileStream.cs:290 
  at System.IO.FileStream..ctor (System.String path, FileMode mode, FileAccess access, FileShare share) [0x00000] in /home/abuild/rpmbuild/BUILD/mono-4.4.2/mcs/class/corlib/System.IO/FileStream.cs:97 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream:.ctor (string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare)
  at System.IO.File.OpenRead (System.String path) [0x00000] in /home/abuild/rpmbuild/BUILD/mono-4.4.2/mcs/class/corlib/System.IO/File.cs:341 
  at System.TimeZoneInfo.FindSystemTimeZoneByFileName (System.String id, System.String filepath) [0x00011] in /home/abuild/rpmbuild/BUILD/mono-4.4.2/mcs/class/corlib/System/TimeZoneInfo.cs:551 
  at System.TimeZoneInfo.CreateLocal () [0x000ed] in /home/abuild/rpmbuild/BUILD/mono-4.4.2/mcs/class/corlib/System/TimeZoneInfo.cs:179 
  at System.TimeZoneInfo.get_Local () [0x0000c] in /home/abuild/rpmbuild/BUILD/mono-4.4.2/mcs/class/corlib/System/TimeZoneInfo.cs:76 
  at System.TimeZoneInfo.GetDateTimeNowUtcOffsetFromUtc (DateTime time, System.Boolean& isAmbiguousLocalDst) [0x00000] in /home/abuild/rpmbuild/BUILD/mono-4.4.2/mcs/class/corlib/System/TimeZoneInfo.cs:491 
  at System.DateTime.get_Now () [0x00008] in /home/abuild/rpmbuild/BUILD/mono-4.4.2/external/referencesource/mscorlib/system/datetime.cs:919 
  at OpenRA.Platforms.Default.MultiTapDetection.<clickHistoryCache>m__1 (Byte _) <0x410db340 + 0x0000b> in <filename unknown>:0 
  at OpenRA.Exts.GetOrAdd[K,V] (System.Collections.Generic.Dictionary`2 d, OpenRA.K k, System.Func`2 createFn) <0x410db010 + 0x0004d> in <filename unknown>:0 
  at OpenRA.Primitives.Cache`2[T,U].get_Item (OpenRA.Primitives.T key) <0x410dafd0 + 0x00023> in <filename unknown>:0 
  at OpenRA.Platforms.Default.MultiTapDetection.DetectFromMouse (Byte button, int2 xy) <0x410daf80 + 0x00023> in <filename unknown>:0 
  at OpenRA.Platforms.Default.Sdl2Input.PumpInput (IInputHandler inputHandler) <0x4108f9e0 + 0x00333> in <filename unknown>:0 
  at OpenRA.Platforms.Default.Sdl2GraphicsDevice.PumpInput (IInputHandler inputHandler) <0x4108f9a0 + 0x0002b> in <filename unknown>:0 
  at OpenRA.Renderer.EndFrame (IInputHandler inputHandler) <0x4108dba0 + 0x00032> in <filename unknown>:0 
  at OpenRA.Game.RenderTick () <0x410d21f0 + 0x003af> in <filename unknown>:0 
  at OpenRA.Game.Loop () <0x410ce810 + 0x001e7> in <filename unknown>:0 
  at OpenRA.Game.Run () <0x410ce6c0 + 0x0006f> in <filename unknown>:0 
  at OpenRA.Program.Run (System.String[] args) <0x40f96aa0 + 0x00053> in <filename unknown>:0 
  at OpenRA.Program.Main (System.String[] args) <0x40f95e00 + 0x000db> in <filename unknown>:0 

```

**Comment by pchote** [comment]
Thanks for the report, but this isn't something we can fix on our end: it's a bug with either your distro's mono packaging or with mono itself (depending on who you ask).

Take a look at #7790 and #11805 for some older discussions and suggested fixes.  The stack traces are a bit different, but they all have the same cause (missing `/etc/localtime` file).

**Comment by Mailaender** [comment]
Please report this as https://bugzilla.opensuse.org/ with a list of installed packages and patterns.
